File: scripts/bench/tax.c
#include <stdio.h>
int main(void) {
    double total = 0.0;
    for (int i = 0; i < 100000; i++) {
        double income = 300000.0 + i;
        double tax = (income - 300000.0) * 0.05;
        total += tax;
    }
    printf("%.2f\n", total);
    return 0;
}

File: scripts/bench/tax.js
// Same computation. JavaScript has no native decimal type, so this is
// double-precision float — fast, but not exact for money.
let total = 0.0;
let i = 0;
while (i < 100000) {
    const income = 300000 + i;
    const tax = (income - 300000) * 0.05;
    total = total + tax;
    i += 1;
}
console.log(total);
